keras/backend/mxnet_backend.py

```diff
@@ -167,62 +126,118 @@ def to_dense(tensor):
     raise NotImplementedError("MXNet Backend: Sparse operations are not supported.")
 
 
-class KerasContext(object):
-    """
-    TODO: Contexts are not yet supported with MXNet backend.
-    """
-    pass
+# class KerasContext(object):
+#     """
+#     TODO: Contexts are not yet supported with MXNet backend.
+#     """
+#     pass
+
+
+def keras_symbol_child(func):
```

Comment:
Can we rename this to something more intuitive - Ex: keras_to_mxnet_symbol_mapper() or something in similar lines?